AjaxControlToolkit/Scripts/AjaxFileUpload.js

```diff
@@ -1136,6 +1154,18 @@ Sys.Extended.UI.AjaxFileUpload.Control.prototype = {
     },
 
     /// <summary>
+    /// Manually starts upload process
+    /// </summary>
+    startUpload: function () {
+        if (this._isUploading || !this._filesInQueue.length) {
+            return;
+        }
+
+        this._onUploadOrCancelButtonClickedHandler();
```

Comment:
Calling `_onUploadOrCancelButtonClickedHandler()` directly starts upload process before `createInputFileElement()` call in `onFileSelectedHandler`:
```js
this.onFileSelectedHandler = function(e) {
	this.addFilesToQueue(e.target.files);
	this.createInputFileElement();
};
```

Change `onFileDroppedHandler()` and `onFileDroppedHandler()` so that `startUpload()` at the end of the methods.